src/test/java/org/threadly/litesockets/buffers/SimpleMergedByteBuffersTests.java

```diff
@@ -0,0 +1,277 @@
+package org.threadly.litesockets.buffers;
+
+import static org.junit.Assert.*;
+
+import java.nio.BufferUnderflowException;
+import java.nio.ByteBuffer;
+import java.util.Random;
+
+import org.junit.After;
+import org.junit.Test;
+import org.threadly.litesockets.buffers.MergedByteBuffers;
+import org.threadly.litesockets.buffers.ReuseableMergedByteBuffers;
+import org.threadly.litesockets.buffers.SimpleMergedByteBuffers;
+
+public class SimpleMergedByteBuffersTests {
```

Comment:
If you want tests can extend classes as well.  I usually have my test class structure closely match the class one to reduce test code duplication.  I can point you to some examples if interested, just DM me.